app/TalkRevision.php

```diff
@@ -30,22 +30,22 @@ public function getUrl()
         return '/talks/' . $this->talk->id . '/?revision=' . $this->id;
     }
 
-    public function getHtmledDescription()
+    public function getDescription()
     {
-        return $this->htmlize($this->getAttribute('description'));
+        return $this->htmlize($this->getAttribute('description'), false);
     }
 
     public function getHtmledOrganizerNotes()
     {
         return $this->htmlize($this->getAttribute('organizer_notes'));
     }
 
-    private function htmlize($string)
+    private function htmlize($string, $changeNewLineToBR = true)
     {
         if ($string == '') {
             return '<i>(empty)</i>';
         }
 
-        return str_replace("\n", "<br>", $string);
+        return $changeNewLineToBR ? str_replace("\n", '<br>', $string) : $string;
```

Comment:
Use the single quotation mark whenever possible.
> "\n"

Reply:
Got it, thanks Jose!